src/main/java/com/easypost/model/beta/ReferralCustomers.java

```diff
@@ -15,12 +15,11 @@
 import java.net.HttpURLConnection;
 import java.net.URL;
 import java.nio.charset.StandardCharsets;
-import java.util.Arrays;
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
 
-public class Referral extends BaseUser {
+public class ReferralCustomers extends BaseUser {
```

Comment:
```suggestion
public class ReferralCustomer extends BaseUser {
```
This class name should be singular